programmers/budget.js

```diff
@@ -0,0 +1,18 @@
+function solution(d, budget) {
+    d.sort((a,b) => {
+        return a-b
+    });
```

Comment:
바로 반환한다면 return을 생략할 수 있습니다.
```suggestion
    d.sort((a,b) => a - b);
```